### PR TITLE
Add web app install docs

### PR DESCRIPTION
**Pull request by carkom**
Adding a section to install documentation that explains how one could use the web app for a quick start without sync.

### CODE REVIEW

File: docs/install/index.md [hunk 1]
```diff
@@ -40,6 +40,20 @@ Once you’ve set up your server, you can [configure it](../config/index.md) to
 
 If you're coming from the original, managed Actual subscription service, you may want to [migrate your data](../migration/index.md).
 
+## Server-Free Web App
+
+If you are okay with not having sync or auto-save or backups then the easiest and fastest way to get started is by using the Actual [web app](https://app.actualbudget.org). There's less setup involved in this solution but more maintenance.
```
Comment (RubenOlsen) [inline docs/install/index.md:45]: - Comma after sync, auto-save, and backups. Remove _or_ between _sync or auto-save_. => _... not having sync, auto-save, or backups, then the ..._
- _There's less setup involved in this solution but more maintenance. _ => What about _This solution requires less setup but more maintenance._?

File: docs/install/index.md [hunk 2]
```diff
@@ -40,6 +40,20 @@ Once you’ve set up your server, you can [configure it](../config/index.md) to
 
 If you're coming from the original, managed Actual subscription service, you may want to [migrate your data](../migration/index.md).
 
+## Server-Free Web App
+
+If you are okay with not having sync or auto-save or backups then the easiest and fastest way to get started is by using the Actual [web app](https://app.actualbudget.org). There's less setup involved in this solution but more maintenance. 
+
+All data is saved to your local browser. The Actual web app never has access to any of your personal data. It is recommended that you save your data (from the [settings](../backup-restore/backup) menu) after every session. If your browser memory is cleared then your data will be lost so having a backup is crucial.
```
Comment (RubenOlsen) [inline docs/install/index.md:47]: - Comma after _cleared_ and _lost_. => _... cleared, your data will be lost, so  ..._

File: docs/install/index.md [hunk 3]
```diff
@@ -40,6 +40,20 @@ Once you’ve set up your server, you can [configure it](../config/index.md) to
 
 If you're coming from the original, managed Actual subscription service, you may want to [migrate your data](../migration/index.md).
 
+## Server-Free Web App
+
+If you are okay with not having sync or auto-save or backups then the easiest and fastest way to get started is by using the Actual [web app](https://app.actualbudget.org). There's less setup involved in this solution but more maintenance. 
+
+All data is saved to your local browser. The Actual web app never has access to any of your personal data. It is recommended that you save your data (from the [settings](../backup-restore/backup) menu) after every session. If your browser memory is cleared then your data will be lost so having a backup is crucial. 
+
+Using a new device or a new browser requires you to [import](../migration/actual-import#import-backup-file) the saved file for each new device or browser. Keep in mind that these new devices and browsers will not sync without a server set up, so anything you modify on one browser will not show up on any others.
```
Comment (RubenOlsen) [inline docs/install/index.md:49]: - Remove _a new_ in _new device or a new browser_.
- Suggestion: Replace _Keep in mind_ with _Remember_.
- Suggestion: Replace _... will not show up on any other._ with _... will not appear on others._

File: docs/install/index.md [hunk 4]
```diff
@@ -40,6 +40,20 @@ Once you’ve set up your server, you can [configure it](../config/index.md) to
...
+
+:::caution
+
+This solution is not recommended for long-term use due to the maintenance required and the high probability of data loss. It's intended as a quick start, if you'd like to continue using Actual long-term, please use one of the server options above.
```
Comment (RubenOlsen) [inline docs/install/index.md:53]: I suggest splitting _It's intended as a ..._ into two sentences, where the 2nd sentence start with _If you'd like...:

**Review by RubenOlsen — COMMENTED** [review]
I like that we have a section on this, but except for my nit-picking on the language I am a bit unsure about the title _Server-Free Web App_. It's a bit of an oxymoron for technically inclined people.

What about a variant of one of the following titles: 

* Using Actual locally in the browser
* Getting started without installing or downloading software

File: docs/install/index.md (hunk 4 above)
Comment (youngcw) [reply docs/install/index.md:53]: @pwinn Its because without a server you can't easily sync between devices or use the built in bank syncing.

**Comment by youngcw** [comment]
It may be good to somehow include that doing this is a fallback option for if setting up a https enabled server is over the users head.  They can use this for PWA but their server.


I have been thinking about maybe making a list of recommended server/client configurations.
Like:
1. PikaPods is super easy, but not free.
2. <mix and match middle options>
3. Host your own with https, hardest but best long term aside from PikaPods

**Comment by MatissJanis** [comment]
cc @carkom just checking in: any plans to pick this up again? If not we can close it to clean up the backlog of PRs.

**Comment by carkom** [comment]
> cc @carkom just checking in: any plans to pick this up again? If not we can close it to clean up the backlog of PRs.

Ready to deploy. Cheers!

**Pull request merged**